subprojects/docs/src/docs/userguide/authoring-builds/ant.adoc

```diff
@@ -84,8 +84,8 @@ To make custom tasks available in your build, you can use the `taskdef` (usually
 
 .Using a custom Ant task
 ====
-include::sample[dir="snippets/ant/useExternalAntTask/groovy",files="build.gradle"]
-include::sample[dir="snippets/ant/useExternalAntTask/kotlin",files="build.gradle.kts"]
+include::sample[dir="snippets/ant/useExternalAntTask/groovy",files="build.gradle[tags=configure-task]"]
```

Comment:
@mlopatkin Btw what the purpose for this change?

Reply:
This is the [asciidoc syntax](https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/) to include only a part of the file. If you look at the changes of `snippets/ant/useExternalAntTask/groovy/build.gradle` you can see that I added some scaffolding code to resolve checkstyle binaries. This new code shouldn't be included in the userguide, so I wrapped the interesting part into tags and added the tag name here.